pkg/sql/catalog/systemschema/system.go

```diff
@@ -149,10 +149,10 @@ const (
   "descID"     INT8,
   version      INT8,
   "nodeID"     INT8,
-  expiration   TIMESTAMP,
+  "sessionID"   BYTES NOT NULL,
   crdb_region  BYTES NOT NULL,
-  CONSTRAINT   "primary" PRIMARY KEY (crdb_region, "descID", version, expiration, "nodeID"),
-  FAMILY       "primary" ("descID", version, "nodeID", expiration, crdb_region)
+  CONSTRAINT   "primary" PRIMARY KEY (crdb_region, "descID",  version, "sessionID", "nodeID"),
+  FAMILY       "primary" ("descID", version, "nodeID", "sessionID", crdb_region)
```

Comment:
should we declare a FK constraint from leaseTable.sessionID to sqllivenessTable.sessionID?

Currently, this is not possible bc sqllivenessTable.sessionID is not unique. Instead, on sqllivenessTable, the primary key is (crdb_region, sessionID). This does make me wonder if we should declare a FK from leaseTable.(crdb_region, sessionID) to sqllivenessTable.(crdb_region, sessionID).

What do you think?